scripts/setup.sh

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
+
 set -e
+
+checkNodeVersion() {
+  runningNodeVersion=$(node -v)
+  requiredNodeVersion=$(cat "$ROOT_DIR/.nvmrc")
+
+  if [ "$runningNodeVersion" != "$requiredNodeVersion" ]; then
+    echo -e "${red}Using wrong version of Node. Required ${requiredNodeVersion}. Running ${runningNodeVersion}.${plain}"
+    exit 1
+  fi
+}
```

Comment:
I don't think this is doing anything at the moment as we're not calling the function. Have made some suggestions below and we probably want to remove `nvm` from the repository too as this change puts the onus on the developer to manage how they change node versions (using `nvm`, `fnm`, `docker` etc.).

```suggestion
BUILD_DIR=$(dirname $0)
ROOT_DIR=$BUILD_DIR/..

checkNodeVersion() {
  runningNodeVersion=$(node -v)
  requiredNodeVersion=$(cat "$ROOT_DIR/.nvmrc")

  if [ "$runningNodeVersion" != "$requiredNodeVersion" ]; then
    echo -e "${red}Using wrong version of Node. Required ${requiredNodeVersion}. Running ${runningNodeVersion}.${plain}"
    exit 1
  fi
}

checkNodeVersion
```

Reply:
Thank you so much @akash1810. Let me know if the latest changes are good to merge 👍🏽